incubator/cassandra/README.md

```diff
@@ -1,60 +1,84 @@
 # Cassandra
-A Cassandra Chart for Kubernetes
 
-## Install Chart
-To install the Cassandra Chart into your Kubernetes cluster (This Chart requires persistent volume by default, you may need to create a storage class before install chart. To create storage class, see [Persist data](#persist_data) section)
+Cassandra is a distributed NoSQL database management system designed to handle
```

Comment:
I agree.  Most of the docs in kubernetes project is 80 (or slightly higher) wrapped.  Is easier to maintian than unwrapped.